package.json

```diff
@@ -37,7 +37,7 @@
     "prop-types": "^15.6.2",
     "query-string": "^4.3.2",
     "radium": "^0.22.1",
-    "ramda": "^0.23.0",
+    "ramda": "^0.26.1",
```

Comment:
I wanted `mergeDeepLeft` so I upgraded `ramda` - but there were two breaking changes there: changed order of args in `adjust`, and removed `import R from 'ramda'` - you can still do `import * as R` but seems like we're moving to explicit imports anyway.